### PR TITLE
Feature/set default template

### PR DESCRIPTION
**Pull request by buttonfreak**
feature(template): allow template to be configured as HTML-string
feature(template): add default template to be used when no config option is defined

### CODE REVIEW

File: src/js/CustomUi.js [hunk 1]
```diff
@@ -90,6 +125,7 @@ class CustomUi extends Meister.Ui {
      * Insert the Ui into the DOM.
      */
     draw() {
+        console.log(this.controlsWrapper, this.element);
```
Comment (FranklinWaller) [inline src/js/CustomUi.js]: CONSOLE LOGGGGG

Comment (Thomasvdam) [inline src/js/CustomUi.js]: This needs to go. :P

Comment (FranklinWaller) [inline src/js/CustomUi.js]: ALREADY DONE THAT

**Comment by buttonfreak** [comment]
@Thomasvdam I rebase faster than my shadow

**Pull request merged**